Engine/Component/ComponentTrail.cpp

```diff
@@ -241,7 +245,8 @@ void ComponentTrail::Render()
 {
 	if (active && trail_texture)
 	{
-
+		glDisable(GL_CULL_FACE);
```

Comment:
Why disabling face culling?

Reply:
I added it because I noticed it was a recommendation from Carlos slides. I can remove it no worries.